spreadsheet.py

```diff
@@ -183,9 +179,10 @@ def append_rows(self, rows, target, first_table_cell="A1"):
         append to happen in the wrong place.
         """
         cell_range = target + "!" + first_table_cell
-        body = { "values": rows }
-        req = self.spreadsheets().values().append(spreadsheetId=self._id, 
-                range=cell_range, valueInputOption='RAW', 
-                insertDataOption='INSERT_ROWS', body=body)        
+        body = {"values": rows}
+        req = self.spreadsheets().values().append(spreadsheetId=self._id,
+                                                  range=cell_range,
+                                                  valueInputOption='RAW',
+                                                  insertDataOption='INSERT_ROWS', # noqa
```

Comment:
That is actually a good trick though, which I didn't know about.   We might want to use that later for things we need to ignore for a short time.